database/src/migrations/release.0.7/create_spatial_extensions.psql

```diff
@@ -0,0 +1,8 @@
+-- create_spatial_extensions.psql
+CREATE EXTENSION IF NOT EXISTS postgis CASCADE;
+CREATE EXTENSION IF NOT EXISTS postgis_raster CASCADE;
+CREATE EXTENSION IF NOT EXISTS postgis_topology CASCADE;
+CREATE EXTENSION IF NOT EXISTS postgis_sfcgal CASCADE;
+CREATE EXTENSION IF NOT EXISTS pgRouting CASCADE;
+CREATE EXTENSION IF NOT EXISTS fuzzystrmatch CASCADE;
+CREATE EXTENSION IF NOT EXISTS pgcrypto CASCADE;
```

Comment:
```suggestion
CREATE EXTENSION IF NOT EXISTS pgcrypto CASCADE;

```

Reply:
either way. same with the rest. feel free to define whats appropriate.

Reply:
I think this was cleared up by formatting and linting?